### PR TITLE
Update 1st responder doc per last week's team mtg

### PR DESCRIPTION
**Pull request by mjgiarlo**
* Tidy the text a bit to be more succinct
* Re-balance the header levels to be a single hierarchy
* Update documentation around FR's queue and honeybadger monitoring responsibilities

### CODE REVIEW

File: infrateam_first_responder.md [hunk 1]
```diff
@@ -37,85 +35,85 @@ PRs to this doc are always welcome, but especially while we're ironing things ou
...
 
-If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it.  List of appropriate places:
-* https://github.com/sul-dlss/DevOpsDocs - e.g. what an ops or devops person would need to know to handle the situation
+If you need to triage or troubleshoot a problem and realize some documentation is missing, please provide it. List of appropriate places:
+* https://github.com/sul-dlss/DevOpsDocs - e.g., what an ops or devops person would need to know to handle the situation
 * README or other top level markdown doc in codebase (viewable via github)
   * Which codebase would need to be apparent from the problem
 * Wiki for the codebase
-* ? - in general, just try to consider where the person interested in the info might look first.  A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs.  Use your best judgement and ask for feedback if unsure.
+* ? - in general, just try to consider where the person interested in the info might look first. A user might go to the wiki, a dev might go to the README, ops folks might head to DevOpsDocs. Use your best judgement and ask for feedback if unsure.
```
Comment (ndushay) [inline infrateam_first_responder.md]: might as well get rid of "just" while you're at it.

File: infrateam_first_responder.md [hunk 2]
```diff
@@ -37,85 +35,85 @@ PRs to this doc are always welcome, but especially while we're ironing things ou
...
+## How to Proactively Check for Production Problems
 
-At the very least, the first responder should be watching incoming alerts from Honeybadger.  (https://app.honeybadger.io/projects)
+At the very least, the first responder should be watching:
 
-Other places to monitor:
-* Nagios alerts (https://sul-nagios-prod.stanford.edu/nagios/  The Services tab on the left column will give overview of checks.  The Problems->Services tab will give you what’s alerting.)
-* feedback email lists, e.g. argo-feedback, sinopia-feedback, etc
+* [Honeybadger alerts](https://app.honeybadger.io/projects) for projects in the Infrastructure portfolio
+  * **Note** that you may want to clear old alerts at the beginning of your first responder shift so you have a clean slate to monitor.
```
Comment (ndushay) [inline infrateam_first_responder.md]: "mark old alerts as resolved" rather than "clear old alerts"?

File: infrateam_first_responder.md [hunk 3]
```diff
@@ -129,7 +127,8 @@ Other places to monitor:
   - `#sdr-operations`
   - `#sul-cap-collab` - has developers in the School of Medicine working on the Profiles project (which we connect to with our sul-pub system)
   - `#web-archiving`
-* Resque dashboards  e.g.
+* Queue dashboards:
+  - **Note** that you may want to clear old failed jobs at the beginning of your first responder shift so you have a clean slate to monitor, but do this deliberately and with caution: clearing failed jobs may prevent investigating current known first responder issues.
```
Comment (ndushay) [inline infrateam_first_responder.md]: Caution here.
robots - we have a sweeper that will rerun stuck jobs (I think?  or only waiting?) and we have workflows that should have the info ... but I'm still a bit unclear if we can just toast all of them without worry.
preservation - I would recommend AGAINST clearing these.  Retrying all connection errors and then clearing those, yes.  Possibly retrying non-connection errors and clearing them is okay.
was-registrar - dunno.
dor-services - we have the procedure for some stuff in the hand off doc - maybe move that here?   Not sure we can/should just "clear" other stuff.

File: infrateam_first_responder.md [hunk 4]
```diff
@@ -1,9 +1,7 @@
 # Guide for Infrastructure Team First Responder
 
-The DLSS infrastructure team is using a rotating role of "first responder".  This doc explains a bit about the concept of
-the first responder role and then outlines specific duties and expectations.
-
-PRs to this doc are always welcome, but especially while we're ironing things out and having regular retrospectives about the process.  If it sticks, and/or the access team adopts it, we can get rid of this caveat.
+The DLSS Infrastructure team is using a rotating role of "first responder." This doc explains the concept of
+the first responder role and outlines specific duties and expectations.
```
Comment (ndushay) [inline infrateam_first_responder.md]: don't do manual line break here;  we don't in other sections.

File: infrateam_first_responder.md (hunk 1 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md]: done.

File: infrateam_first_responder.md (hunk 2 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md]: went with "resolve old alerts in the Honeybadger UI" 

thanks!

File: infrateam_first_responder.md (hunk 3 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md]: I'm going to remove it altogether.

File: infrateam_first_responder.md (hunk 4 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md]: That was my editor "helping" me. :roll_eyes: Fixed!

**Pull request merged**